docs/requirements.txt

```diff
@@ -2,3 +2,4 @@ recommonmark==0.7.1
 # Pin this to the same major version as https://docs.python.org/3/
 Sphinx==2.4.4  # pyup: <3.0
 sphinx-argparse==0.2.5
+docutils==0.16  # https://github.com/sphinx-doc/sphinx/issues/9049
```

Comment:
This is what actually fixes this, and could be fixed without getting rid of Docker. But I started in the other end and don't think it's worth splitting the PR.